src/test/java/org/kohsuke/github/GHContentIntegrationTest.java

```diff
@@ -158,4 +160,11 @@ public void testMIMELonger() throws IOException {
                 + "123456789012345678901234567890123456789012345678901234567890");
         ghContentBuilder.commit();
     }
+
+    @Test
+    public void testGetFileContentWithNonAsciiPath() throws Exception {
+        final GHContent fileContent = gitHub.getRepository("hub4j-test-org/GHContentIntegrationTest")
+                .getFileContent("ghcontent-ro/a-file-with-\u00F6");
+        assertThat(IOUtils.readLines(fileContent.read(), StandardCharsets.UTF_8), hasItems("test"));
```

Comment:
I'm looking for a test something like this (I haven't run locally): 

```suggestion
        final GHRepository repo = gitHub.getRepository("hub4j-test-org/GHContentIntegrationTest");
        final GHContent fileContent = repo.getFileContent("ghcontent-ro/a-file-with-\u00F6");
        assertThat(IOUtils.readLines(fileContent.read(), StandardCharsets.UTF_8), hasItems("test"));
        
        final GHContent fileContent2 = repo.getFileContent(fileContent.getPath());
        assertThat(IOUtils.readLines(fileContent2.read(), StandardCharsets.UTF_8), hasItems("test"));
```

Reply:
I just checked; your modification works on my machine. So yes, please go with it.